service/controller/cluster.go

```diff
@@ -269,6 +271,7 @@ func NewCluster(config ClusterConfig) (*Cluster, error) {
 	{
 		c := v15.ClusterResourceSetConfig{
 			CertsSearcher:      config.CertsSearcher,
+			G8sClient:          config.G8sClient,
```

Comment:
Had to configure and backport the `statusresource` to make it work consistently. Below you will find the same changes for `v14`, `v14patch1` and ? v15`.